webapp/src/pages/profileList/components/ProfileListComponent.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from "react";
+import { withRouter} from "react-router";
+ import ReactTable from 'react-table';
+import { profileService} from "../../../services/profilelist";
+
+const DEFAULT_EVENT_ID = process.env.REACT_APP_DEFAULT_EVENT_ID || 1;
+
+class ProfileListComponent extends Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            List:[],
+            isEmpty:true,
+            loading:true,
+            error: ""
+        }
+}
+
+ isListEmpty(){
+
+    return this.state.List.length === 0;
+ }
+
+ componentDidMount(){ 
+    this.setState({List:profileService.getProfilesList(DEFAULT_EVENT_ID)
```

Comment:
This outer setState isn't necessary

Reply:
When I console logged my List, it firsts prints undefined, [], then undefined, but it does display the data. But I have removed the setState as per request